.mypy.ini

```diff
@@ -20,6 +20,7 @@ warn_unused_ignores = True
 disallow_any_unimported = True
 warn_return_any = True
 exclude = .*_pb2.py
+ignore_missing_imports = True
```

Comment:
we usually apply it for that specific package.. like `scipy`, `adb_shell` below ..

Reply:
I was getting this error from mypy

`error: Skipping analyzing "semantic_version": module is installed, but missing library stubs or py.typed marker [import-untyped]`

Reply:
he means you apply it on the specific package thats triggering the mypy error, check other agents for reference.